openchain-blockchain-coinprism/azuredeploy.parameters.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
@Flavien Can you look at the CI guide to enable this parameter file. Thanks!
https://github.com/Azure/azure-quickstart-templates/tree/master/1-CONTRIBUTION-GUIDE#travis-ci